docs/k8s-quickstart.asciidoc

```diff
@@ -230,7 +230,7 @@ Use `kubectl port-forward` to access Kibana from your local workstation:
 kubectl port-forward service/quickstart-kb-http 5601
 ----
 +
-Open `http://localhost:5601` in your browser.
+Open `https://localhost:5601` in your browser.
```

Comment:
Should we mention a note about the SSL browser warning? ` For testing purposes only, bypass the warning, the self-signed certificate configured by default is not verified by a third party certificate authority and not trusted by your browser.`?

Reply:
Maybe: `Your browser will show a warning because the self-signed certificate configured by default is not verified by a third party certificate authority and not trusted by your browser. You can either configure a (link)[valid certificate] or acknowledge the warning for the purposes of this quick start`